### ISSUE TITLE
incompatible gem version

### ISSUE THREAD

**Issue by eidosabi** [kristianmandrup/cantango#1]
When I try to `bundle` cantango into a rails 3.1.0.rc4 app I get the following:

```
Bundler could not find compatible versions for gem "arel":
  In Gemfile:
    cantango depends on
      arel (~> 2.0.7)

    rails (= 3.1.0.rc4) depends on
      arel (2.1.3)

```

**Comment by kristianmandrup** [comment]
OK, thanks! I will fix that ASAP. It turns out it is due to _meta_where_ dependency. It will be removed and made optional.

https://github.com/ryanb/cancan/wiki/MetaWhere

```
meta_where (1.0.4)
  activerecord (~> 3.0.0)
  activesupport (~> 3.0.0)
  arel (~> 2.0.7)
```

Released new patched version 0.8.2.1

**Comment by eidosabi** [comment]
Thank you!
